dashboard/package.json

```diff
@@ -7,7 +7,7 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "start:docker": "next dev -p 9876",
+    "start:docker": "node server.js",
```

Comment:
Where does this `server.js` file come from? I don't see it copied into the Dockerfile nor do I see anything to generate it?

Reply:
the next.js `standalone` mode generates a package with everything we need and it is just a node app.

Reply:
Yeah, but I don't see that file being copied over in the `Dockerfile`?

Reply:
We are copying the whole folder https://github.com/iasql/iasql/pull/2043/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R107

Reply:
Ohh... I didn't notice that `standalone` was getting copied into the `/dashboard` root.